assets/app.js

```diff
@@ -5,39 +5,23 @@
  * (and its CSS file) in your base layout (base.html.twig).
  */
 
-// any CSS you import will output into a single css file (app.css in this case)
 import './styles/app.css';
 import 'iconify-icon';
 import './styles/global.scss';
 import './styles/basic_styles.css';
 import 'bootstrap/scss/bootstrap.scss';
+import 'jquery-ui/themes/base/autocomplete.css';
+import 'jquery-bootstrap-theme/css/custom-theme/jquery-ui-1.10.3.custom.css';
 
-//import '~/bootstrap/scss/bootstrap.scss';
 const $ = require('jquery');
 global.$ = global.jQuery = $;
 
-//am4art
-/*import './js/amcharts/core';
-import './js/amcharts/charts';
-import './js/amcharts/wordCloud';
-import './js/amcharts/moonrisekingdom';
```

Comment:
why did you delete charts ?

Reply:
> why did you delete charts ?

It was COMMENTED rudiment import, before I installed them via NPM